packages/eslint-plugin-template/src/rules/i18n.ts

```diff
@@ -422,7 +422,7 @@ function isBooleanLike(value: string): value is 'false' | 'true' {
 }
 
 function isEmpty(value: string) {
-  return value.trim().length === 0;
+  return value == null || value.trim().length === 0;
```

Comment:
Would it be worth to have something like the following?
```js
if (!!value) {
  return value.trim().length === 0;
}

return true;
```

Reply:
I suppose that could save us some unnecessary calls to ```trim```, so it might be slightly more performant.